modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/utils/dataUtils.ts

```diff
@@ -102,6 +104,43 @@ export function hasDataChanged({
 	return false;
 }
 
+export function hasEndpointDataChanged({
+	fetchedEndpointData,
+	localUIData,
+}: {
+	fetchedEndpointData: APIEndpointItem;
+	localUIData: Partial<APIEndpointUIData>;
+}) {
+	if (
```

Comment:
@cgmonte You could join all those `if`s inside just one using `||`. The performance will be the same and the code will become leaner.